docs/en/guides/testing-SFCs-with-jest.md

```diff
@@ -49,7 +49,7 @@ Next, create a `jest` block in `package.json`:
       // process `*.vue` files with `vue-jest`
       ".*\\.(vue)$": "<rootDir>/node_modules/vue-jest"
     },
-    "mapCoverage": true
+    "collectCoverage": true
```

Comment:
Can you remove collectCoverage from here, since the previous example wasn't collecting coverage

Reply:
@eddyerburgh Oh, OK!